source/features/keyboard-navigation.tsx

```diff
@@ -4,6 +4,30 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import {isEditable} from '../helpers/dom-utils';
 
+function isCommentGroupMinimized(comment: HTMLElement): boolean {
+	if (select.exists('.minimized-comment:not(.d-none)', comment)) {
+		return true;
+	}
+
+	if (comment.closest('.js-resolvable-thread-contents.d-none')) {
+		return true;
+	}
+
+	if (comment.closest('.js-resolvable-timeline-thread-container:not([open])')) {
```

Comment:
Merge into `comment.closest` above?

Reply:
True, the function can look something like:

```js
return (
	exists('0') ||
	closest(`
		1,
		2
	`)
);
```